### DISCUSSION TITLE
孩子们玄学真的有用

### DISCUSSION THREAD

**Discussion by feixiao090822** [share121/inter-knot#5482] [原神]
![IMG_20240930_155946](https://github.com/user-attachments/assets/12300caa-c1aa-4790-b9ca-46dbacc35fa8)
![IMG_20240930_155845](https://github.com/user-attachments/assets/d98b9f6e-b53f-4931-85e2-4d362b202df2)
![IMG_20240930_155725](https://github.com/user-attachments/assets/a66e59ba-e6e8-4890-903d-1f6a3c4fa95d)
![Uploading IMG_20240930_155651.jpg…]()

**Comment by aomengxin** [comment]
我是来看这个的吗（）
